test/integration/app_test.exs

```diff
@@ -0,0 +1,67 @@
+defmodule Integration.AppTest do
```

Comment:
I'm not sure I. Doctests are for ensuring the correctness of generated documentation. Using it exclusively for integration tests would be unweildy, especially since the documentation in this repo is going to be dev docs, rather than user docs, which will live in the reference repo. https://github.com/slime-lang/slime-lang.github.io